### PR TITLE
fix(home-manager/kitty): use new `themeFile` option on 24.11

### PR DESCRIPTION
**Pull request by olifloof**
also removed kitty from the IFD section in the faq, as `themeFile` doesn't use IFD

### CODE REVIEW

**Comment by olifloof** [comment]
also note: this requires the `master` branch of home-manager

**Comment by olifloof** [comment]
i should probably find a way to make it work for older versions

File: modules/home-manager/kitty.nix [hunk 1]
```diff
@@ -7,5 +7,5 @@ in
 {
   options.programs.kitty.catppuccin = ctp.mkCatppuccinOpt { name = "kitty"; };
 
-  config.programs.kitty = lib.mkIf enable { theme = "Catppuccin-${ctp.mkUpper cfg.flavor}"; };
+  config.programs.kitty = lib.mkIf enable { themeFile = "Catppuccin-${ctp.mkUpper cfg.flavor}"; };
```
Comment (khaneliman) [inline modules/home-manager/kitty.nix:10]: This should be a path, I'd assume? You are still just passing the name of a theme.

**Comment by getchoo** [comment]
Waiting on #306 to pull in a newer revision of home-manager for CI

**Pull request merged**